types/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "fastify/types/tsconfig.json"
```

Comment:
Looks like it works, but i think we should create basic config in the **root** directory of project and then use it in all plugins. It will enforce to unify plugin project structures and provide type checking in all IDEs.
The file is currently located in the /types directory, but it includes files from /types and /test/types, which is a bit strange.

Reply:
I think it is better to leave it in types anyway: Fastify is written in JS and the types are just an addition. Putting it in the root could trick VS code (and probably other IDEs) to treat it like a TS project.